File: driver-service/src/main/java/io/voitovich/yura/driverservice/properties/DefaultKafkaProperties.java
package io.voitovich.yura.driverservice.properties;

import lombok.Data;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.stereotype.Component;

@ConfigurationProperties(prefix = "default.kafka")
@Component
@Data
public class DefaultKafkaProperties{
    private String confirmRatingReceiveTopicName;
    private String consumeRatingTopicName;
}

File: ride-service/src/main/kotlin/io/voitovich/yura/rideservice/properties/DefaultKafkaProperties.kt
package io.voitovich.yura.rideservice.properties

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.stereotype.Component


@ConfigurationProperties(prefix = "default.kafka")
@Component
class DefaultKafkaProperties {
    var sendRatingToDriverTopicName = ""
    var sendRatingToPassengerTopicName = ""
    var sendRatingToDriverTopicReplicasAssignments: Short = 1
    var sendRatingToDriverTopicNumPartitions = 1
    var sendRatingToPassengerTopicReplicasAssignments: Short = 1
    var sendRatingToPassengerTopicNumPartitions = 1
}